vic/drivers/image/src/get_global_param.c

```diff
@@ -1140,21 +1170,6 @@ get_global_param(FILE *gp)
         log_err("ROOT_ZONES must be defined to a positive integer greater "
                 "than 0, in the global control file.");
     }
-    if (options.LAI_SRC == LAI_FROM_VEGPARAM && !options.VEGPARAM_LAI) {
-        log_err("\"LAI_SRC\" was specified as \"LAI_FROM_VEGPARAM\", but "
-                "\"VEGPARAM_LAI\" was set to \"FALSE\" in the global "
-                "parameter file.  If you want VIC to read LAI values from "
-                "the vegparam file, you MUST make sure the veg param file "
-                "contains 1 line of 12 monthly LAI values for EACH veg "
-                "tile in EACH grid cell, and you MUST specify "
-                "\"VEGPARAM_LAI\" as \"TRUE\" in the global parameter "
-                "file.  Alternatively, if you want VIC to read LAI values "
-                "from the veg library file, set \"LAI_SRC\" to "
-                "\"LAI_FROM_VEGLIB\" in the global parameter file.  In "
-                "either case, the setting of \"VEGPARAM_LAI\" must be "
-                "consistent with the contents of the veg param file (i.e. "
-                "whether or not it contains LAI values).");
-    }
```

Comment:
Actually, I think we should remove some more of these checks in this particular file, since netcdf i/o functions called by vic_init() and vic_force() will handle attempts to read variables that don't exist.

However, your point *does* apply to the classic driver.  I agree, I will see if there are any other cases to handle.